users/models.py

```diff
@@ -0,0 +1,97 @@
+from django.db import models
+from django.utils import timezone
+from django.core.validators import validate_email, RegexValidator
+from django.contrib.auth.models import make_password
+from django.contrib.auth.models import (
+    AbstractBaseUser, BaseUserManager, PermissionsMixin)
+from django.conf import settings
+
+from common.models import County, AbstractBase
+
+
+USER_MODEL = settings.AUTH_USER_MODEL
+
+
+class MflUserManager(BaseUserManager):
+    def create_user(self, email, first_name,
+                    username, password=None, **extra_fields):
+        now = timezone.now()
+        if not email:
+            raise ValueError('The email must be set')
+        validate_email(email)
+        p = make_password(password)
+        email = MflUserManager.normalize_email(email)
+        user = self.model(email=email, first_name=first_name, password=p,
+                          username=username,
+                          is_staff=False, is_active=True, is_superuser=False,
+                          last_login=now, date_joined=now, **extra_fields)
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(self, email, first_name, username,
+                         password, **extra_fields):
+        user = self.create_user(email, first_name,
+                                username, password, **extra_fields)
+        user.is_staff = True
+        user.is_active = True
+        user.is_superuser = True
+        user.save(using=self._db)
+        return user
+
+
+class MflUser(AbstractBaseUser, PermissionsMixin):
```

Comment:
Class naming conventions...where "MFL" is an abbreviation, the class should be named "MFLUser".